JuliaExperimental/gap/singular.g

```diff
@@ -497,7 +501,7 @@ InstallOtherMethod( \*,
                  Julia.Base.("*")( x, JuliaPointer( y ) ) );
     end );
 
-#T Calling 'divexact' causes a segmentation fault in Julia!
+#T Calling 'divexact' directly causes a segmentation fault in Julia!
```

Comment:
Could you please clarify this? I.e., what invocation exactly crashes? Perhaps insert it as part of the comment?

Reply:
@fingolfin Thanks for the hint.
In fact, now (with a new Julia and a new Singular) the "natural" way works.